kolibri/content/models.py

```diff
@@ -287,29 +269,13 @@ class ChannelMetadataAbstractBase(models.Model):
     author = models.CharField(max_length=400, blank=True)
     version = models.IntegerField(default=0)
     thumbnail = models.TextField(blank=True)
-    root_pk = UUIDField()
-
-    class Meta:
-        abstract = True
-
-    def __str__(self):
-        return self.name
-
-
-class ChannelMetadata(ChannelMetadataAbstractBase, ContentDatabaseModel):
-    """
-    This class stores the channel metadata within the content database itself.
-    """
-
-    objects = ContentQuerySet.as_manager()
-
-
-class ChannelMetadataCache(ChannelMetadataAbstractBase):
-    """
-    This class stores the channel metadata cached/denormed into the primary database.
-    """
-
     last_updated = DateTimeTzField(null=True)
+    # Minimum version of Kolibri that this content database is compatible with
+    min_kolibri_version = models.CharField(max_length=50)
```

Comment:
It would come from the `CONTENT_SCHEMA_VERSION` imported from the `kolibri.content.models` that are used during the export process. I suppose I could even set that as the default value to save the CC Server having to explicitly set it.